src/test/scala/net/ripe/rpki/publicationserver/store/postgresql/PgStoreTest.scala

```diff
@@ -328,4 +328,74 @@ class PgStoreTest extends PublicationServerBaseTest with Hashing {
     ))
   }
 
+  test("should allow same object to be published in multiple locations") {
+    val clientId1 = ClientId("client1")
+    val clientId2 = ClientId("client2")
```

Comment:
ClientId2 looks unused in this test, maybe it was meant to be used in PublishQ for uri2.  Also when the URI differs for the same hash, wasn't this supposed to be because they are from a different clients?